tests/unit/test-extension-runners/test.chromium.js

```diff
@@ -47,9 +47,9 @@ function prepareExtensionRunnerParams({ params } = {}) {
 
 describe('util/extension-runners/chromium', async () => {
   it('uses the expected chrome flags', () => {
-    // Flags from chrome-launcher v0.14.0
+    // Flags from chrome-launcher v1.0.0
     const expectedFlags = [
-      '--disable-features=Translate',
+      '--disable-features=Translate,OptimizationHints,MediaRouter',
```

Comment:
@willdurand this PR needed this set of expected flags to be updated and so I had to push an additional commit to the PR as originally created by dependabot.